### ISSUE TITLE
Add option for making the unique microsend-etc data a containing folder for file

### ISSUE THREAD

**Issue by bglusman** [CargoSense/briefly#22]
Right now the filename is made unique with a timestamp-derived string appended after a prefix(if any), and while the file clearly needs to have a unique path, it would be nice if the temp file itself didn't necessarily have to have such a unique/meaningless name.

Putting a more explicitly defined filename into a unique directory similarly named via microsecond level timestamp/system unique integer etc should allow this and save the user from having to rename the file somehow before presenting a fully controlled filename to a user.

**Comment by bglusman** [comment]
Just randomly noticed this old issue and had to remind myself, but, since inactive almost 3 years, would a PR for this be welcome I guess?  not sure if/when i'd get around to, but, curious/tempted...  @warmwaffles  maybe just to ping someone who's been more recently active in the project?

**Comment by warmwaffles** [comment]
Unfortunately I don't think this project is actively maintained anymore. I've poked @benwilson512 a few times in slack to see if we can get a new version published, but I think the original author is busy elsewhere and doesn't want to maintain it.

I wouldn't mind maintaining it to just keep the lights on, but I don't know if that's what is desired.

**Comment by mcrumm** [comment]
Hi folks, I'm back "in the office" next week. We'll follow up issues and work on getting a new release out then, thanks!

**Comment by warmwaffles** [comment]
@mcrumm 🥳 yay

**Comment by warmwaffles** [comment]
Any updates @mcrumm

**Comment by mcrumm** [comment]
@warmwaffles I just released v0.4.0 on hex :) https://hex.pm/packages/briefly/0.4.0

@bglusman I think you want the `:directory` option:

```elixir
{:ok, dir} = Briefly.create(directory: true)

File.stat!(dir).type
# => :directory
```
